mkt/extensions/validation.py

```diff
@@ -19,6 +19,8 @@ class ExtensionValidator(object):
     error.
     """
     errors = {
+        'AUTHOR_TOO_LONG': _(
+            u'The `author` property cannot be longer than 255 characters.'),
```

Comment:
Device side UX would prefer 128 and that seems enough to me.  Can we change to 128?